src/Calculator.Shared/Styles.xaml

```diff
@@ -178,1610 +177,1617 @@
-                      Foreground="{TemplateBinding Foreground}"
-                      BorderBrush="{ThemeResource AppControlForegroundTransparentRevealBorderBrush}"
-                      BorderThickness="{TemplateBinding BorderThickness}"
-                      AutomationProperties.AccessibilityView="Raw"
-                      Content="{TemplateBinding Content}"
-                      ContentTemplate="{TemplateBinding ContentTemplate}"
-                      ContentTransitions="{TemplateBinding ContentTransitions}" />
-          </Grid>
-        </ControlTemplate>
-      </Setter.Value>
-    </Setter>
-  </Style>
-
-  <Style x:Key="SquareIconButtonStyle"
-       TargetType="Button">
-
-    <Setter Property="Foreground"
-        Value="{ThemeResource SystemControlForegroundBaseHighBrush}" />
-    <Setter Property="Background"
-        Value="{ThemeResource SystemControlBackgroundTransparentBrush}" />
-    <Setter Property="BorderThickness"
-        Value="1" />
-    <Setter Property="Margin"
-        Value="0" />
-    <Setter Property="Padding"
-        Value="0" />
-    <Setter Property="Width"
-        Value="{StaticResource HamburgerHeight}" />
-    <Setter Property="Height"
-        Value="{StaticResource HamburgerHeight}" />
-    <Setter Property="MinHeight"
-        Value="16" />
-    <Setter Property="MinWidth"
-        Value="16" />
-    <Setter Property="VerticalAlignment"
-        Value="Stretch" />
-    <Setter Property="HorizontalAlignment"
-        Value="Stretch" />
-    <Setter Property="HorizontalContentAlignment"
-        Value="Center" />
-    <Setter Property="VerticalContentAlignment"
-        Value="Center" />
-    <Setter Property="FontFamily"
-        Value="{StaticResource CalculatorFontFamily}" />
-    <Setter Property="FontSize"
-        Value="16" />
-
-    <Setter Property="Template">
-      <Setter.Value>
-        <ControlTemplate TargetType="Button">
-
-          <Grid x:Name="RootGrid"
-              Background="{TemplateBinding Background}">
-
-            <VisualStateManager.VisualStateGroups>
-              <VisualStateGroup x:Name="CommonStates">
-                <VisualState x:Name="Normal" />
-                <VisualState x:Name="PointerOver">
-                  <VisualState.Setters>
-                    <Setter Target="RootGrid.(RevealBrush.State)"
-                        Value="PointerOver" />
-                    <Setter Target="RootGrid.Background"
-                        Value="{ThemeResource AppControlHoverButtonFaceBrush}" />
-                    <Setter Target="ContentPresenter.Foreground"
-                        Value="{ThemeResource SystemControlHighlightAltBaseHighBrush}" />
-                  </VisualState.Setters>
-                </VisualState>
-                <VisualState x:Name="Pressed">
-                  <VisualState.Setters>
-                    <Setter Target="RootGrid.(RevealBrush.State)"
-                        Value="Pressed" />
-                    
```

Comment:
These missing `ThemeResource` values should probably be added to `SystemResources.xaml` in Uno, can you create a [GitHub issue](https://github.com/nventive/Uno/issues) to track this?

Reply:
Most of those are in a PR that is not merged yet.